lms/djangoapps/mobile_api/users/views.py

```diff
@@ -258,35 +259,43 @@ class UserCourseEnrollmentsList(generics.ListAPIView):
           certified).
         * url: URL to the downloadable version of the certificate, if exists.
     """
-    queryset = CourseEnrollment.objects.all()
-    serializer_class = CourseEnrollmentSerializer
-    lookup_field = 'username'
-
-    # In Django Rest Framework v3, there is a default pagination
-    # class that transmutes the response data into a dictionary
-    # with pagination information.  The original response data (a list)
-    # is stored in a "results" value of the dictionary.
-    # For backwards compatibility with the existing API, we disable
-    # the default behavior by setting the pagination_class to None.
-    pagination_class = None
```

Comment:
Ah.  If they aren't using pagination, then yes, this won't be an issue.  Thanks for the cleanup.  
Did you verify for both Android and iOS?

Ideally, it's better to use pagination to be a good bandwidth citizen.  That would require updating the mobile app to support, for example, "infinite scrolling" (so as the user scrolls down, more pages appear).  However, that course dashboard view needs to be updated regardless. The UI itself doesn't scale for a large number of courses, without being able to sort/search/classify/etc.